src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -75,7 +75,7 @@ public bool ProcessCoverageReports(ILogger logger)
                 if (TryGetTrxFiles(settings, out var trxPaths) &&
                     trxPaths.Any())
                 {
-                    File.AppendAllText(propertiesFilePath, $"{SonarProperties.VsTestReportsPaths}={string.Join(",", trxPaths.Select(c => c.Replace(@"\", @"\\")))}");
+                    File.AppendAllText(propertiesFilePath, $"{Environment.NewLine}{SonarProperties.VsTestReportsPaths}={string.Join(",", trxPaths.Select(c => c.Replace(@"\", @"\\")))}");
```

Comment:
the properties file is generated without an empty line at the end. Due to this, we have to insert a new line before writing the properties